aiida/tools/importexport/dbexport/utils.py

```diff
@@ -270,3 +281,32 @@ def check_process_nodes_sealed(nodes):
             'All ProcessNodes must be sealed before they can be exported. '
             'Node(s) with PK(s): {} is/are not sealed.'.format(', '.join(str(pk) for pk in nodes - sealed_nodes))
         )
+
+
+def summary(file_format, outfile, **kwargs):
```

Comment:
to be consistent, this should also be printed to the logger, right?

Reply:
It would be good to have this information in there as well, yes.